Src/BootCamp.Chapter/Demo/DemoDelegatesAndEvents.cs

```diff
@@ -0,0 +1,55 @@
+﻿using BootCamp.Chapter.Events;
+using System;
+using System.Transactions;
+
+namespace BootCamp.Chapter.Demo
+{
+    public static class DemoDelegatesAndEvents
+    {
+        private static event EventHandler<ApplicationStateEventArgs> ApplicationStateHandler;
+        private static event EventHandler<InputPressedEventArgs> InputHandler;
+
+        public static void Run()
+        {
+            // Subscribe for events so we can catch input presses and application state changes
+            ApplicationStateHandler += ApplicationStateEvents.OnApplicationStateChange;
```

Comment:
Normally, it should be subscribed to from the outside. If it's nested like that, it could simply be a normal method.

Reply:
Indeed. This should be fixed now.